<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Part3</class>
 <widget class="QWidget" name="Part3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblTitleP3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Find gender of the person by using GridSearch and CrossValidation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTestP3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadClassifierP3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Classifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSectionGammeP3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>sectionGamma:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editMinGammeP3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-4</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatusP3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSectionGammeP3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinGammeP3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>minGamma:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTestOnImgP3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test On Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTrainP3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveClassifierP3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Classifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadDataP3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDisplayP3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxGammeP3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>maxGamma:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editMaxGammeP3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxCP3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>maxC:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinCP3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>minC:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editMaxCP3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSectionCP3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>sectionC:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSectionCP3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>520</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editMinCP3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFoldsP3">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>510</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Folds:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFoldsP3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>510</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
